pkg/sources/github/github.go

```diff
@@ -970,13 +989,9 @@ func (s *Source) addOrgsByUser(ctx context.Context, user string) {
 		}
 		logger.V(2).Info("Listed orgs", "page", orgOpts.Page, "last_page", resp.LastPage)
 		for _, org := range orgs {
-			var name string
-			if org.Login != nil {
-				name = *org.Login
-			} else {
+			if org.Login == nil {
 				continue
 			}
-			common.AddStringSliceItem(name, &s.orgs)
```

Comment:
what replaced this deleted call?

Reply:
same as above, it was replaced with the cache.